### PR TITLE
Assessment 06

### PR DESCRIPTION
**Pull request by RuthGichaba**

### CODE REVIEW

File: rcttc-queries.sql [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+where `date` between '2021-10-01' and '2021-12-31'; -- 3 shows
+
+-- 2. List customers without duplication.
+select concat(first_name, ' ', last_name) as `name`
+from customer; -- 74 rows
+
+-- 3. Find all customers without a `.com` email address.
+select * 
+from customer
+where email like '%.com'; -- 42 rows
```
Comment (jmccann-dev-10) [inline rcttc-queries.sql:16]: I think this is giving back the customers WITH a .com

File: rcttc-queries.sql [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+-- 12. Who is the biggest supporter of RCTTC? Who spent the most in 2021?
+select 
+	r.customer_id,
+    count(reservation_id) as `tickets purchased`, 
+    s.ticket_price,
+    count(reservation_id) * s.ticket_price as `total revenue`
+from reservation r
+inner join shows s on r.show_id = s.show_id
+group by r.customer_id, s.ticket_price;
```
Comment (jmccann-dev-10) [inline rcttc-queries.sql:103]: The aggregates on these are giving back incorrect data.  The show total should be across all nights performed and not total per evening.

File: rcttc-schema.sql [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    constraint fk_reservation_theater_id
+		foreign key (theater_id)
+        references theater(theater_id),     
+    constraint fk_reservation_show_id
+		foreign key (show_id)
+		references shows(show_id),
+	constraint fk_reservation_customer_id
+		foreign key (customer_id)
+		references customer(customer_id)   
+);
```
Comment (jmccann-dev-10) [inline rcttc-schema.sql:48]: These look really good, although some of the data is still repeated per table.

**Review by jmccann-dev-10 — CHANGES_REQUESTED** [review]
Pretty good start so far, there are a few queries that need to be cleaned up because they are currently giving back incorrect data, but otherwise good job.  I really like how well organized the queries are.

File: rcttc-queries.sql (hunk 1 above)
Comment (RuthGichaba) [reply rcttc-queries.sql]: Fixed it!

File: rcttc-queries.sql (hunk 2 above)
Comment (RuthGichaba) [reply rcttc-queries.sql]: Should be correct now!

File: rcttc-schema.sql (hunk 3 above)
Comment (RuthGichaba) [reply rcttc-schema.sql]: Adjusted the schema for shows so theater info is not repeated now~

**Review by jmccann-dev-10 — APPROVED** [review]
Excellent job, keep up the good work!

**Pull request merged**